docs/application/tizen-studio/rt-ide/getting-started/flash.md

```diff
@@ -2,38 +2,32 @@
 
 To upload your project on the board, you need to flash it:
 
-1. Connect the board to your computer.
-
-   The following figure shows the Artik 051 board.
-
-   ![Connecting the board](media/rt_flash_connect.png)
+1. Connect the Artik 053 board to your computer:
+ 
+     ![Connecting the board](media/rt_flash_connect.png)
 
 2. Select the project in the **Project Explorer** view.
 
-3. To flash the Tizen RT project, use one of the following:
+3. To flash the TizenRT project, use one of the following:
 
    - In the Tizen Studio for RT menu, select **Project > Flash**.
-   - In the Tizen Studio for RT toolbar, click the **Flash** icon (![Flash icon](media/rt_icon_flash.png)).
+   - In the Tizen Studio for RT toolbar, click the (![Flash icon](media/rt_icon_flash.png)):
 
      ![Flashing the project](media/rt_flash.png)
 
-   If you have not built the project yet, an error popup appears.
+   If you have not built the project yet, an error popup appears:
 
-   ![Flashing without building error](media/rt_flash_build_error.png)
+     ![Flashing without building error](media/rt_flash_build_error.png)
 
-4. In the Flash TizenRT Project Wizard, select the Flash option and click **Flash**.
+4. In the Flash TizenRT Project Wizard, select the **Flash** option, and click **OK**:
```

Comment:
Question. Is this update checked with the actual Tizen Studio? Why the bold font is applied to the **Flash**?

Reply:
@mijin85cho, Thank you for pointing it out. I will remove bold tag.